src/main/webapp/itempagescript.js

```diff
@@ -132,18 +132,20 @@ function deleteComment(commentId) {
  *     timestamp
  * @param { boolean } belongsToUser - boolean that shows whether or not the
  *     current user has posted a comment
- @param { string } commentId - unique Id which references the specific comment.
+ * @param { string } email - the email of the user who posted the comment
  * @returns { html element }  returns a list element
- */ 
-function createListElement(comment, belongsToUser, commentId) {
+ */
+function createListElement(comment, belongsToUser, email) {
   const liElement = $('<li class="list-group-item"></li>');
   liElement.text(comment);
   if (belongsToUser) {
     liElement.append(
         $('<div onclick="deleteComment(' + commentId +
           ')"><i class="fa fa-trash" style="float:right;"></i></div>'));
   }
-
+  liElement.append(
+      $('<img class="pr-1" src="' + getProfileImageUrl(email) + '.png?s=23'+
```

Comment:
What is the significance of ```?s=23```? Should it be added to the getProfileImageUrl() function or no?

Reply:
That is just the size of the image so that it's small enough to be an icon. It should not be in the getProfileImageUrl() function